supabase/migrations/091_unify_event_rooms.sql

```
@@ -0,0 +1,380 @@
+-- 4) room_experiment_runs / room_experiment_visits / room_exit_survey_responses:
+--    放送回ルーム検証(room-experiments)をイベントルームにも対応させる room_kind 判別列を追加
+
+-- ================================================================
+-- 1) posts.event_id
+-- ================================================================
+ALTER TABLE public.posts
+    ADD COLUMN IF NOT EXISTS event_id uuid REFERENCES public.events(id) ON DELETE SET NULL;
+
+CREATE INDEX IF NOT EXISTS posts_event_id_idx
+    ON public.posts (event_id, created_at DESC)
+    WHERE event_id IS NOT NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)